packages/ubuntu_bootstrap/lib/pages/confirm/confirm_page.dart

```diff
@@ -4,13 +4,19 @@ import 'package:flutter_riverpod/flutter_riverpod.dart';
 import 'package:subiquity_client/subiquity_client.dart';
 import 'package:ubuntu_bootstrap/l10n.dart';
 import 'package:ubuntu_bootstrap/pages/confirm/confirm_model.dart';
+import 'package:ubuntu_provision/interfaces.dart';
 import 'package:ubuntu_wizard/ubuntu_wizard.dart';
 import 'package:yaru_widgets/yaru_widgets.dart';
 
-class ConfirmPage extends ConsumerWidget {
+class ConfirmPage extends ConsumerWidget with ProvisioningPage {
   const ConfirmPage({super.key});
 
-  static Future<bool> load(WidgetRef ref) {
+  @override
+  Future<bool> load(BuildContext context, WidgetRef ref) {
+    return ref.read(confirmModelProvider).init().then((_) => true);
+  }
+
+  static Future<bool> loadStatic(WidgetRef ref) {
```

Comment:
Is there a use case for this, or is it just a leftover from debugging?

Reply:
Removed!